modules/ROOT/pages/dataweave-types-coercion.adoc

```diff
@@ -236,10 +235,13 @@ output application/json
 [source,json,linenums]
 ----
 {
-  "data": 22.00
+  "data": "22.00"
 }
 ----
 
+[NOTE]
+To maintain consistency across all formats, starting with DataWeave 2.5, writing number values directly doesn't preserve formatting and strips trailing zeroes. Format metadata is only used to parse the provided string into the correct number representation. To retain this behavior, you can re-enable it by using the `stripTrailingZeroes` property. See xref:dataweave-system-properties.adoc[supported data formats].
```

Comment:
Since you're referring to the section, can you leave out the hard-coded text and let antora fill it in?